Buildings/ThermalZones/EnergyPlus/BaseClasses/SpawnExternalObject.mo

```diff
@@ -1,7 +1,7 @@
 within Buildings.ThermalZones.EnergyPlus.BaseClasses;
 class SpawnExternalObject
   "Class used to couple the FMU to interact with a thermal zone"
-  //extends Modelica.Icons.BasesPackage;
+  extends Modelica.Icons.BasesPackage;
```

Comment:
This didn't work in OpenModelica since the specification says that classes extending from ExternalObject are not allowed to contain any other elements, and we interpreted that in the strictest sense (extends clauses are technically elements). But this seems like something that should be allowed, so I've relaxed our check to allow extends clauses as long as they don't contain any components or classes.

Reply:
Thanks for the feedback, I will remove the `extends` through https://github.com/lbl-srg/modelica-buildings/pull/2386

Reply:
My comment was mostly just to tell you that you don't have to change anything though, but if you wish to remove the extends anyway you're of course free to do so.

Reply:
I understand, and I rather stay compliant with the standard as this `extends` is not important.